pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -1392,131 +1609,1374 @@ def marker_minimum(self):
+        on, a dashed line indicating the squelch level appears on the display.
+        A marker must be active and above the squelch line for demodulation to occur. Refer to
+        the :attr:`demodulation_mode` command. The default value is -120 dBm.
+
+        Type: :code:`str,int`
+
+        .. code-block:: python
+
+            instr.preset()
+            instr.start_frequency = 88e6
+            instr.stop_frequency = 108e6
+
+            instr.peak_search(PeakSearchMode.High)
+            instr.demodulation_time = 10
+
+            instr.squelch = -60
+            instr.demodulation_mode = DemodulationMode.FM
+        """,
+        validator=joined_validators(strict_discrete_set, strict_range),
+        values=[["ON", "OFF"], range(-220, 30)],
+        set_process=lambda v: v if isinstance(v, str) else v
+    )
+
+    def service_request(self, input):
+        """Triggers a service request. This command allows you to force a
+        service request and test a program designed to handle service requests.
+        However, the service request can be triggered only if it is first
+        masked using the :attr:`request_service_conditions` command.
+
+        :param input: Bits to emulate a service request
+        :type input: :class:`StatusRegister`
+        """
+        if input not in range(0, 255):
+            raise ValueError("Bit mask needs to be between 0 ... 255")
+
+        self.write("SRQ %d" % input)
+
+    # `center_frequency_step_size` would be a command but is pretty unnecesary
+
+    sweep_time = Instrument.control(
+        "ST?", "ST %s",
+        """
+        Control the sweep time. This is normally a coupled function which is
+        automatically set to the optimum value allowed by the current instrument settings.
+        Alternatively, you may specify the sweep time. Note that when the specified sweep time is
+        too fast for the current instrument settings, the instrument is no longer calibrated and the
+        message 'MEAS UNCAL' appears on the display. The sweep time cannot be adjusted when the
+        resolution bandwidth is set to 10 Hz, 30 Hz, or 100 Hz.
+
+        Type: :code:`str, float`
+
+        Real from 50E—3 to 100 when the span is greater than 0 Hz; 50E—6 to 60 when
+        the span equals 0 Hz. When the resolution bandwidth is <100 Hz, the sweep time
+        cannot be adjusted.
+        """,
+        validator=joined_validators(strict_discrete_set, strict_range),
+        values=[["AUTO", "MAN"], arange(50E-6, 100)],
+        set_process=lambda v: v if isinstance(v, str) else ("%.3E" % v)
+    )
+
+    status = Instrument.measurement(
+        "STB?",
+        """
+        Returns to the controller the decimal equivalent of the bits set in the
+        status byte (see the RQS and SRQ commands). STB is equivalent to a serial poll command.
+        The RQS and associated bits are cleared in the same way that a serial poll command would
+        clear them.
+        """,
```

Comment:
Please start with "Measure" or "Get"
```suggestion
        Get the decimal equivalent of the bits set in the
        status byte (see the RQS and SRQ commands). STB is equivalent to a serial poll command.
        The RQS and associated bits are cleared in the same way that a serial poll command would
        clear them.
        """,
```